okteto.yml

```diff
@@ -0,0 +1,19 @@
+name: java-maven-getting-started
+build:
+  java-maven-getting-started:
+    context: .
+    dockerfile: Dockerfile
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/maven:3
+    command: java -jar *.jar
+    workdir: /app
+    sync:
+      - .:/app
```

Comment:
`.:/usr/src/app`

Reply:
? should I also update the Dockerfile then, 
the Dockerfile is staging the code to WORKDIR /app

Reply:
no, we are using `okteto/maven:3`, which is not built by that Dockerfile